src/components/MarkdownProvider/index.tsx

```diff
@@ -59,12 +61,19 @@ export const MarkdownProvider: React.FC = ({ children }) => (
     <GlobalStyle />
     <MDXProvider
       components={{
+        /**
+         * Garden components
+         */
+        MD,
```

Comment:
We shouldn't be providing `MD` components directly. Even in JSX/HTML code we should use `p, 2-h6, etc.` which gets mapped to the corresponding styling given by this provider. 

This is especially important to avoid `<div>` and `<span>` usage. Since this is text content we need the semantic meaning of the elements.

Reply:
This should be a `<b>` and not `MD`.